### PR TITLE
Memory parameter singleCPUMem is now possible to use

### PR DESCRIPTION
**Pull request by alneberg**
The `--singleCPUMem` parameter was not possible to use on the commandline as it was interpretted as a string instead of a memory unit. This PR fixes this problem.

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [x] `CHANGELOG.md` is updated

### CODE REVIEW

**Comment by MaxUlysse** [comment]
Should we check if similar issues arise with `--max_memory`, `--max_cpus`, `--max_time`?

**Comment by MaxUlysse** [comment]
No need to fix this line?
https://github.com/nf-core/sarek/blob/34e871cd4a7cfe84c5b67e3b2b673b47dabf110b/conf/base.config#L26

File: conf/base.config [hunk 1]
```diff
@@ -8,6 +8,7 @@
  * on the PATH. Runs in `local` mode - all jobs will be
  * run on the logged in environment.
  */
+import nextflow.util.MemoryUnit
```
Comment (MaxUlysse) [inline conf/base.config]: Is this needed?
We have that in the `nextflow.config` file, but no import:
https://github.com/nf-core/sarek/blob/ca2a0b0e41128f86d24ab683074592ca26b81df8/nextflow.config#L112-L125

Comment (alneberg) [reply conf/base.config]: Ah, let me check, perhaps it was added during debugging.

Comment (MaxUlysse) [reply conf/base.config]: Can you check my other comments in the conversation as well?

Comment (alneberg) [reply conf/base.config]: I got an error when removing it:
```
ERROR ~ Unable to parse config file: '/lupus/proj/ngi2016004/private/johannes/src/sarek_dev/nextflow.config'

  Compile failed for sources FixedSetSources[name='/groovy/script/ScriptAFC21C7604A8062E1F8130921F0D5B8E']. Cause: org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
  /groovy/script/ScriptAFC21C7604A8062E1F8130921F0D5B8E: 53: unable to resolve class MemoryUnit
   @ line 53, column 30.
         memory = {check_resource((params.singleCPUMem as MemoryUnit) * 2 * task.attempt)}
                                  ^

  /groovy/script/ScriptAFC21C7604A8062E1F8130921F0D5B8E: 56: unable to resolve class MemoryUnit
   @ line 56, column 30.
         memory = {check_resource((params.singleCPUMem as MemoryUnit) * task.attempt * task.attempt)}
                                  ^

  2 errors


 -- Check '.nextflow.log' file for details
```
So I don't think that it works.

Comment (MaxUlysse) [reply conf/base.config]: Is it needed in `nextflow.config` then?

**Pull request merged**